File: idakonyvtar/src/main/java/eu/eyan/idakonyvtar/model/LibraryModel.java
package eu.eyan.idakonyvtar.model;

import com.jgoodies.binding.list.SelectionInList;

public class LibraryModel {
	private SelectionInList<Book> books = new SelectionInList<Book>();

	private Library library;

	public SelectionInList<Book> getBooks() {
		return books;
	}

	public Library getLibrary() {
		return library;
	}

	public void setLibrary(Library library) {
		this.library = library;
	}

}

File: idakonyvtar/src/main/scala/eu/eyan/idakonyvtar/model/LibraryModel.scala
package eu.eyan.idakonyvtar.model;

import com.jgoodies.binding.list.SelectionInList;

class LibraryModel(
    val books: SelectionInList[Book] = new SelectionInList[Book](),
    var library: Library = null) /* var, because of loading other library */
